__tests__/app.test.js

```diff
@@ -12,25 +14,40 @@ beforeEach(() => {
   return seed(data);
 });
 
-describe('/api/topics', () => {
-  test('GET: 200 Get all topics', () => {
-    return request(app).get("/api/topics")
-    .expect(200)
-    .then(({body}) => {
-      const {topics} = body;
+describe("/api/topics", () => {
+  test("GET: 200 Get all topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
 
-      expect(Array.isArray(topics)).toBe(true)
-      topics.forEach((topic) => {
-        expect(typeof topic.slug).toBe('string')
-        expect(typeof topic.description).toBe('string')
-      })
+        expect(Array.isArray(topics)).toBe(true);
```

Comment:
You can check the length matches what we expect before your forEach to avoid getting any false positives 👍